neo-cli/neo-cli.csproj

```
@@ -28,7 +28,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.0-CI00909" />
+    <PackageReference Include="Neo" Version="3.0.0-CI00921" />
```

```suggestion
    <PackageReference Include="Neo" Version="3.0.0-CI00924" />
```

@joeqian10  Plz, update neo version, the ut bug was fixed in neo core.